spec/index.bs

```diff
@@ -3522,8 +3526,125 @@ The following validation rules apply:
             [=mipmap level=] |textureCopyView|.{{GPUTextureCopyView/mipLevel}}.
 </div>
 
+## Debug Markers ## {#command-encoder-debug-markers}
+
+Command encoders provide methods to apply debug labels to groups of commands or insert a single
+label into the command sequence. Debug groups can be nested to create a hierarchy of labeled
+commands. These labels MAY be passed to the native API backends for tooling, MAY be used by the user
```

Comment:
let's not capitalize MAY in the spec, perhaps?

Reply:
Happy to oblige, but I will note that it's a little odd for a W3C spec to avoid using the RFC2119 keywords.

Reply:
I don't mind the keyword itself, but having it capitalized is just not consistent with the rest of the spec. We can talk about capitalizing them separately from this PR.

Reply:
👍 FYI: One instance of this slipped through previously in the Shader Module Creation section. Want me to adjust the casing on that while I'm at it?